### PR TITLE
Feat(footer): Add social links

### PR DESCRIPTION
**Pull request by lawlesx**
Solved Issue - #94 

Updated the footer as per the design provided

Screenshot 2024-08-31 235216
Screenshot 2024-08-31 235228

### CODE REVIEW

File: src/components/footer.tsx
Comment (Jay-Karia) [inline src/components/footer.tsx:1]: For best practices,
The links array should be declared outside of JSX.

```
const links = [...]
```

Comment (lawlesx) [reply src/components/footer.tsx:1]: Yeah, Sorry did it in a hurry.
Pushed a refactor.
Please take a look.

**Review by Jay-Karia — CHANGES_REQUESTED** [review]
The variable declared should be before the `return` statement

**Comment by lawlesx** [comment]
😅 why is that? It's not dependent on anything so we can separate it from
the component.
Is some build failing?

**Comment by Jay-Karia** [comment]
Take any JSX/TSX file (https://github.com/zen-browser/www/blob/main/src/components/marketplace.tsx)
The variables are always declared above the the return statement, however it does not change the result, but as mentioned earlier (for best practices).

Even the original code (with `lists` variable inside the jsx) did not broke the code.

**Comment by lawlesx** [comment]
> Take any JSX/TSX file (https://github.com/zen-browser/www/blob/main/src/components/marketplace.tsx) The variables are always declared above the the return statement, however it does not change the result, but as mentioned earlier (for best practices).
> 
> Even the original code (with `lists` variable inside the jsx) did not broke the code.

I moved the social links to a separate file so later we can add more and the changes will reflect automatically

**Review by Jay-Karia — APPROVED** [review]

**Comment by lawlesx** [comment]
@Jay-Karia Also added a `.vscode` config to remove unused import it will reduce the unnecessary import in the bundle

**Comment by mauro-balades** [comment]
Could we do a rounded icon for discord? It looks out of place

**Comment by mauro-balades** [comment]
Missclicked

**Comment by mauro-balades** [comment]
Sorry, but i've changed the footer's design... but thanks!

**Comment by lawlesx** [comment]
> Sorry, but i've changed the footer's design... but thanks!

Was there a new design?

**Pull request closed without merging**